ecf/subseasonal/stats/jevs_cfs_subseasonal_grid2grid_stats.ecf

```diff
@@ -78,13 +83,13 @@ export cfs_ver=${cfs_ver}
 #export obsproc_ver=${obsproc_ver}
 export VERIF_CASE=grid2grid
 
-export COMROOT=/lfs/h2/emc/vpppg/noscrub/$USER
+export COMROOT=/lfs/h2/emc/vpppg/noscrub/$USER/evs_subseasonal_test/dev/com
 #export COMIN=$COMROOT/$NET/$evs_ver
-export COMIN=/lfs/h2/emc/vpppg/noscrub/$USER/$NET/$evs_ver/prep/$COMPONENT/$RUN
-export COMINobs=/lfs/h2/emc/vpppg/noscrub/$USER/$NET/$evs_ver/prep/$COMPONENT/$RUN
+export COMIN=/lfs/h2/emc/vpppg/noscrub/shannon.shields/EVS_Data/$NET/$evs_ver/prep/$COMPONENT/$RUN
+export COMINobs=/lfs/h2/emc/vpppg/noscrub/shannon.shields/EVS_Data/$NET/$evs_ver/prep/$COMPONENT/$RUN
 export FIXevs=/lfs/h2/emc/vpppg/noscrub/emc.vpppg/verification/EVS_fix
 export COMINclimo=$FIXevs/climos/atmos
-export COMOUT=/lfs/h2/emc/vpppg/noscrub/$USER/$NET/$evs_ver/$STEP/$COMPONENT
+export COMOUT=/lfs/h2/emc/vpppg/noscrub/$USER/evs_subseasonal_test/dev/com/$NET/$evs_ver/$STEP/$COMPONENT
 #export VDATE=$(date -d "today -26 day" +"%Y%m%d")
 
```

Comment:
Hi, @ShannonShields-NOAA ,

Before I merge into the parallel, I wanted to draw your attention to some of these changes that were made in the ecf files.  Particularly, lines 83 and 84 in red above were changed to lines 88 and 89 in green.  There are other changes made throughout the ecf scripts that were used for testing.  However, they should be changed back to their original lines for parallel usage.  

Can we go through the ecf files to ensure that the various COMROOT, COMIN, COMOUT, etc, are properly set to the way they were for the parallel before?  We may need to submit a new PR for this.  @ShelleyMelchior-NOAA added for awareness.

Thanks!

Perry

Reply:
@PerryShafran-NOAA Ah yes, I see what you are saying; sorry for missing that! I will see what I can do within GitHub to correct those ecf lines and submit a new PR.